tests/skipped_tests_gpu.tbl

```diff
@@ -27,9 +27,7 @@ tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-nancumprod-data11]
 tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-nancumsum-data12]
 tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-nanprod-data13]
 tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-nansum-data14]
-tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-negative-data15]
 tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-prod-data16]
-tests/test_sycl_queue.py::test_1in_1out[level_zero:gpu:0-sign-data17]
```

Comment:
could you please check if we can unmute `tests/third_party/cupy/math_tests/test_misc.py::TestMisc::test_sign_negative`?

Reply:
This test includes complex numbers for which `dpnp` falls back and `numpy` and `NotImplementedError` is raised unless environment variable is set to `DPNP_RAISE_EXCEPION_ON_NUMPY_FALLBACK` to `0` if the fall back is required to be supported without raising an exception.

Reply:
Ok, I see. Then we can leave the test muted util the full suport of complex type is implemented for `dpnp.sign`